roles/agent/defaults/main.yml

```diff
@@ -11,8 +11,14 @@ agent_state: present
 agent_version: latest
 # Port for the Beszel binary agent to listen on
 agent_port: 45876
+# Authentication method 1: Public key + Individual system token (traditional method)
 # Public key used to authenticate the Beszel binary agent to the Hub
 agent_public_key: ""
+# Token for individual system authentication
+agent_token: ""
+# Authentication method 2: Universal token (Beszel v0.12.0+)
+# Universal token for automatic agent registration (no public key needed)
+agent_universal_token: ""
```

Comment:
Hi @MohamedYassin-J 

Just looking at this again, technically the role did already support the universal token. The `agent_token` variable could be the token from the agent registration or the Universal Token.

I think this breaking change would be OK as long as we make sure the user cannot specify both `agent_token` and `agent_universal_token` at the same time, I made a comment on this PR about that.

Thanks again for your contribution so far. 🙂